libs/utils/utils.js

```diff
@@ -1192,13 +1192,15 @@ async function loadPostLCP(config) {
 
   const georouting = getMetadata('georouting') || config.geoRouting;
   if (georouting === 'on') {
-    const { default: loadGeoRouting } = await import('../features/georoutingv2/georoutingv2.js');
-    await loadGeoRouting(config, createTag, getMetadata, loadBlock, loadStyle);
+    import('../features/georoutingv2/georoutingv2.js')
```

Comment:
Given @vhargrave concerns in the discussion, I suppose this was brought up with the georouting PM?